lessons/07-mcp/README.md

```diff
@@ -0,0 +1,482 @@
+# MCP, Model Context Protocol
+
+In this chapter you will learn:
+
+- How to use the Model Context Protocol, MCP to split up your server capabilities in a server and a client
+- Create tools, resources and prompts on the server.
+- Consume the server using both the inspector tool and a written client.
+
+## Setup
+
+If you haven't already, set up your development environment. Here's how you can do it: [Setup your environment](/docs/setup/README.md).
+
+## Related resources
+
+[![Watch a short video about MCP](https://img.youtube.com/vi/YRfOiB0Im64/0.jpg)](https://www.youtube.com/watch?v=YRfOiB0Im64)
+
+_This video explains Model Context Protocol._
```

Comment:
Looks like the other chapters also have a slide deck link in addition to the video. Do we have a deck that can be added here (guessing so)?